drop-transfer/src/ws/events.rs

```diff
@@ -246,6 +258,10 @@ impl FileEventTx<IncomingTransfer> {
         .await
     }
 
+    pub async fn preflight(&self) {
+        self.preflight_inner().await
```

Comment:
The `fn preflight_inner()` is used only once here. Can you inline it?

Reply:
This requires making `preflight_inner()` or equivalent to be public which I don't really like

Reply:
Um, I don't get what you mean. My point was: that since `_inner` is used only once (here), then we don't need it as a separate function. Is it used somewhere else as well?

Reply:
Oh I get it now. Misunderstood at first. Inlined it